DOC-4296: News documentation US creation 04 and 05 (#106)
exoplatform/exo-documentation: docs/GettingStarted.rst

.. note:: The |image157| button is grey and unclickable until the mandatory fields *title* and *content* are filled. 

         

> Sharing a news from complete form

Reviewer: ```suggestion
Sharing a news from full form
```

----------------------------------

> To share an article from the full form, you have to display the small form as explained in the previous paragraph and click on "More options" button.

Reviewer 2: I would add a sentence to explain what is the full form and why the user would want to use it, something like "A more detailed form is available to post a news in case you want to add a summary and an illustration.".

> To share an article from the complete form, you have to display the short form as explained in the previous paragraph and click on "More options" button.

Reviewer: ```suggestion
To share an article from the full form, you have to display the small form as explained in the previous paragraph and click on "More options" button.
```

|image159|

> The displayed complete form page contains :

Reviewer: ```suggestion
The full form contains :
```

> - A **Title** field :  Allows you to enter the news title. The title should not exceed 150 characters: Beyond that limit you will not be allowed to write. This field is mandatory.

Reviewer 2: Not sure it is necessary to describe these fields again here, they are already described in the previous chapter. We should only describe the new fields IMO.

> - A **Summary** field : Allows you to enter the news summary. It is optional and will be inserted after the title in the news preview. 

Reviewer 2: We should mention that in case the summary is not filled, the news preview displays the content.

> - A **Summary** field : Allows you to enter the news summary. It is optional and will be inserted after the title in the news preview. 

Reviewer 2: The summary is limited to 1000 characters.

> - A **Summary** field : Allows you to enter the news summary. The summary if inserted will be displayed in the news preview. This field is optional.

Reviewer: ```suggestion
- A **Summary** field : Allows you to enter the news summary. It is optional and will be inserted after the title in the news preview. 
```

> - An "Upload image" area: Allows you to upload optionally an image as a illustrative vignette of the article.

Reviewer 2: Shouldn't we add the rules for the image (size, type) ?
